scripts/sanity_chk/expr_parser.py

```diff
@@ -10,6 +10,15 @@
 import threading
 import re
 
+ZEPHYR_BASE = os.environ.get("ZEPHYR_BASE")
+if not ZEPHYR_BASE:
+    sys.stderr.write("$ZEPHYR_BASE environment variable undefined.\n")
+    exit(1)
+
+sys.path.insert(0, os.path.join(ZEPHYR_BASE, "scripts", "dts"))
+
+import edtlib
```

Comment:
I'm not clear on how this works, for the code in here that does things like `edt.nodes` do we not need edtlib?

Reply:
No need, since it gets passed as an argument.

Only modules that directly look up things like `edtlib.EDT` need to import it.

All `import foo` really does in Python is to load the module and assign it to the variable `foo` (there's a `module` class that represents modules). Python doesn't have declarations or anything like that.